pkg/cluster/cluster.go

```diff
@@ -268,16 +268,27 @@ func (c *Cluster) run(stopC <-chan struct{}) {
 				c.status.Control()
 			}
 
-			running, pending, err := c.pollPods()
+			ready, unready, err := c.pollPods()
 			if err != nil {
 				c.logger.Errorf("fail to poll pods: %v", err)
 				continue
 			}
-			if len(pending) > 0 {
-				c.logger.Infof("skip reconciliation: running (%v), pending (%v)", k8sutil.GetPodNames(running), k8sutil.GetPodNames(pending))
+			var readyMembers []string
+			for _, pod := range ready {
+				readyMembers = append(readyMembers, pod.GetName())
+			}
+			var unreadyMembers []string
+			for _, pod := range unready {
+				unreadyMembers = append(unreadyMembers, pod.GetName())
+			}
+			c.status.ReadyMembers = readyMembers
+			c.status.UnreadyMembers = unreadyMembers
```

Comment:
We will never set `UnreadyMembers` due to skipping logic. Need to resolve it in this PR